doc/org.eclipse.cdt.doc.user/src/asciidoc/example.adoc

```diff
@@ -0,0 +1,8 @@
+ifdef::env-github[]
+:imagesdir: https://raw.githubusercontent.com/eclipse-cdt/cdt/main/doc/org.eclipse.cdt.doc.user/images
```

Comment:
This is a github-specific directive to locate images in the specified location for the puspose of rendering documents in github itself. Note that new images added by a PR won't yet exist at the required location. Example: https://github.com/jld01/cdt/blob/adoc-example/doc/org.eclipse.cdt.doc.user/src/asciidoc/example.adoc